functions.php

```diff
@@ -153,7 +153,7 @@ function wp_auth0_url_base64_decode( $input ) {
 		$padlen = 4 - $remainder;
 		$input .= str_repeat( '=', $padlen );
 	}
-	return base64_decode( strtr( $input, '-_', '+/' ) );
+	return base64_decode( strtr( $input, '-_', '+/' ) ) ?: false;
```

Comment:
why is this needed, and what would happen if the caller receives false? The method doc says it returns a string.

Reply:
You might be looking at the method above it, wp_auth0_url_base64_encode; wp_auth0_url_base64_decode returns [either a bool or a string](https://github.com/auth0/wp-auth0/blob/dd1c1268d50b2fdd20e32cc46fc2896fddd1df40/functions.php#L146). base64_decode will return false when it's passed improperly formatted data. However I should have applied this rule with an extra parameter here, rather than a ternary. Fixed in next commit.